index.js

```diff
@@ -115,9 +116,20 @@ io.on('connection', socket => {
 
   socket.on('message', (content) => {
     io.to(gameID).emit('anon message', {
+      sender: playerNickname,
       content,
       playerID: socket.id
     })
+    if (content.includes("/nick ")) {
+      const tempPlayerNickname = content.substring(5).trim()
+      const regex = new RegExp("^([a-zA-Z0-9_]{1,15})$")
```

Comment:
We cannot trust the client.

Reply:
Can you explain more the issue ?

Reply:
Never mind. I thought this was on the client for a second. My bad.